### PR TITLE
Add Sentry support

### PR DESCRIPTION
**Pull request by fridex**
Depends-On: https://github.com/thoth-station/core/pull/150

### CODE REVIEW

**Comment by fridex** [comment]
I was not able to update lock (some local issues), let's use Kebechet for that.

File: thoth/common/logging.py [hunk 1]
```diff
@@ -85,9 +88,16 @@ def init_logging(logging_configuration: dict = None) -> None:
 
     _init_log_levels(logging_configuration)
 
+    if _SENTRY_DSN:
```
Comment (goern) [inline thoth/common/logging.py]: any docs on this? what and why to do?

File: Pipfile [hunk 2]
```diff
@@ -7,6 +7,7 @@ name = "pypi"
 "rfc5424-logging-handler" = "*"
 daiquiri = "*"
 requests = "*"
+raven = "*"
```
Comment (goern) [inline Pipfile]: why raven? it reads like "Raven is the legacy Python client for Sentry (getsentry.com) — replaced by sentry-python https://sentry.io"

Comment (fridex) [reply Pipfile]: Ah, they are fast. That is the way I did it in the previous project :)

File: thoth/common/logging.py (hunk 1 above)
Comment (fridex) [reply thoth/common/logging.py]: In the core repo.

**Pull request merged**